SegM8/clear/patch.xodp

```diff
@@ -4,7 +4,7 @@
     {
       "description": "Triggers a new clear.",
       "id": "BJkYMOrbv",
-      "label": "DO",
```

Comment:
The pins were renamed, but the C++ implementation is not updated.
Things like `isInputDirty<input_DO>(ctx)` won't compile.

Reply:
Thanks! I will fix it. And will add 'any' to q-s-n

Reply:
Done